src/components/BookForm.js

```diff
@@ -0,0 +1,21 @@
+import './BookForm.css';
+
+const BookForm = () => (
+  <div className="book-form-wrapper">
+    <div>
+      <div>Add New Book</div>
+      <form className="book-form">
+        <input placeholder="Book title" className="book-title" />
```

Comment:
- [ ] It should render a form with two inputs for the title and the author of the book